src/Common/CIPipeline.Types.ts

```diff
@@ -314,8 +314,16 @@ export enum InputOutputVariablesHeaderKeys {
     VALUE = 'val',
 }
 
+export enum ConditionDataTableHeaderKeys {
+    VARIABLE = 'variable',
+    OPERATOR = 'operator',
+    VALUE = 'val',
+}
+
 export type InputOutputVariablesErrorObj = Record<InputOutputVariablesHeaderKeys, DynamicDataTableCellValidationState>
 
+export type ConditionDetailsErrorObj = Record<ConditionDataTableHeaderKeys, DynamicDataTableCellValidationState>
```

Comment:
We should not expport this if not getting used anywhere. Also can we create a type `conditionDetails` and `isConditionDetailsValid` both so that it can be reused